docker-compose.yml

```diff
@@ -35,6 +35,14 @@
 # 4. When finished using delfin project, bring down containers using following command
 #
 #    $ docker-compose down
+#
+# 5. To bring up delfin project with multiple service instances
+#
+#    $ docker-compose up -d --scale <<service-name>>=<<number of instances>>
+#
+#    example: Deploy delfin with 3 delfin-task and 2 delfin-alert instances
```

Comment:
How an example for below use case 
 3 nodes 
1st node ( api 1, task 1, alert 1)
2nd node (task 1)
3rd node (task 1)

Reply:
The PR only handle usecase of installing multiple instance of delfin components in single host.